Change log entry for EPL Apps Util.formatTime [PAM-35326] (#3681)
Cumulocity-IoT/c8y-docs: content/change-logs/analytics/apama-in-c8y-20250829-Cumulocity_Utils_FormatTime.md

---
date:

> title: EPL Apps 'Util.formatTime' will correctly produce date-time string in ISO8601 format

Reviewer: ```suggestion
title: EPL Apps utility Util.formatTime produces date-time string in ISO8601 format
```

change_type:
  - value: change-3BQrQ6adS
    label: API change
product_area: Analytics
component:
  - value: component-M5-cepIIS
    label: Streaming Analytics
build_artifact:
  - value: tc-KXXmo2SUR
    label: apama-in-c8y
ticket: PAM-35326
version: 26.199.0
---

> `Util.formatTime` **behavior changed** in the Cumulocity Utilities bundle for EPL Apps. This utility now produces a date-time string in **ISO8601 format** (`yyyy-MM-dd'T'HH:mm:ss.SSS`). This is a **breaking change** for any code relying on the previous `yyyy/MM/dd HH:mm:ss` format. We recommend updating your code to handle the new standard format.

Reviewer: ```suggestion
The behavior of the `Util.formatTime` has changed in the {{< product-c8y-iot >}} Utilities bundle for EPL Apps. This utility now produces a date-time string in **ISO8601 format** (`yyyy-MM-dd'T'HH:mm:ss.SSS`). This is a **breaking change** for any code relying on the previous `yyyy/MM/dd HH:mm:ss` format. We recommend updating your code to handle the new standard format.
```

> The behavior of the `Util.formatTime` has changed in the {{< product-c8y-iot >}} Utilities bundle for EPL Apps. This utility now produces a date-time string in **ISO8601 format** (`yyyy-MM-dd'T'HH:mm:ss.SSS`). This is a **breaking change** for any code relying on the previous `yyyy/MM/dd HH:mm:ss` format. We recommend updating your code to handle the new standard format.

Reviewer: ```suggestion
The behavior of the `Util.formatTime` has changed in the {{< product-c8y-iot >}} utilities bundle for EPL Apps. This utility now produces a date-time string in **ISO8601 format** (`yyyy-MM-dd'T'HH:mm:ss.SSS`). This is a **breaking change** for any code relying on the previous `yyyy/MM/dd HH:mm:ss` format. We recommend updating your code to handle the new standard format.
```
